frontend/__tests__/components/chat/chat-input.test.tsx

```diff
@@ -171,9 +171,9 @@ describe("ChatInput", () => {
     // Fire paste event with text data
     fireEvent.paste(input!, {
       clipboardData: {
-        getData: (type: string) => type === 'text/plain' ? 'test paste' : '',
-        files: []
-      }
+        getData: (type: string) => (type === "text/plain" ? "test paste" : ""),
+        files: [],
```

Comment:
Is this normal, from '' to "" ? I thought our precommit makes it the other way around.

Reply:
Yes, the [Prettier default for quotes](https://prettier.io/docs/en/options.html#quotes) are double quotes instead of single. We're [not overriding this rule](https://github.com/All-Hands-AI/OpenHands/blob/main/frontend/.prettierrc.json#L1-L3) either so there shouldn't be a reason our pre-commit converts them to single quotes 🤔